tests/library/test_track.py

```diff
@@ -492,3 +492,12 @@ def test_disc_sort(self):
         track2 = MetaTrack(album=MetaAlbum(title="a"), track_num=1, disc=2)
 
         assert track1 < track2
+
+
+class TestDurationProperty:
+    """Test duration property functionality."""
+
+    def test_duration_property_access(self):
```

Comment:
This should be moved under the `TestProperties` class to be grouped with the rest of the property tests.